part-spec/adc_dac.json

```diff
@@ -0,0 +1,187 @@
+{
+    "$id": "https://github.com/chromeos/digital-datasheets/blob/main/part-spec/adc_dac.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "adc_dac",
+    "oneOf": [
+        {
+            "title": "adc",
+            "type": "object",
+            "required": [
+                "componentID"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "digitalResolution": {
+                    "description": "number of bits of resolution in the digital output",
+                    "type": "number"
+                },
+                "conversionTime": {
+                    "description": "time required to convert from an analog signal to digital output",
+                    "comment": "units of seconds",
+                    "$ref": "definitions.json#/unit"
+                },
+                "sampleRate": {
+                    "description": "maximum rate at which the ADC can convert samples",
+                    "$comment": "unit of hertz",
+                    "$ref": "definitions.json#/unit"
+                },
+                "offsetError": {
+                    "description": "difference (in LSB) of the output at the zero point between an actual and ideal ADC",
+                    "comment": "units of LSB",
+                    "type": "number"
+                },
+                "gainError": {
+                    "description": "difference (in LSB) of how the actual transfer function matches the ideal transfer function, also called full scale error",
+                    "comment": "units of LSB",
+                    "type": "number"
+                },
+                "integralNonlinearity": {
+                    "description": "deviation of an actual transfer function from an ideal transfer function, in LSB",
+                    "comment": "units of LSB",
+                    "type": "number"
+                },
+                "differentialNonlinearity": {
+                    "description": "difference (in LSB) in step width between the actual and ideal transfer functions",
+                    "comment": "units of +/- LSB",
+                    "type": "number"
+                },
+                "rmsNoise": {
+                    "description": "root mean square (RMS) noise of ADC",
+                    "comment": "units of LSB",
+                    "type": "number"
+                },
+                "SNR": {
+                    "description": "signal to noise (SNR) ratio of the converter",
+                    "$ref": "definitions.json#/ratio"
```

Comment:
SNR is dB. You can add "units of dB" in "comments" and have it as a value.

Reply:
Ah good point. (I blame sleep deprivation on that mistake...)